contrib/automation_tests/test_cases/live_tab.py

```diff
@@ -87,3 +87,20 @@ def _execute(self, function_name, min_calls=0, max_calls=pow(2, 33)-1):
         self.expect_true(min_calls <= call_count <= max_calls,
                          'Call count for function "%s" expected to be between %s and %s, was %s'
                          % (function_name, min_calls, max_calls, call_count))
+                         
+class VerifyOneFunctionWasCalled(LiveTabTestCase):
+    """
+    Verify that at least one of the functions matching the function name has 
+    received the given number of hits.
+    """
+    def _execute(self, function_name, min_calls=0, max_calls=pow(2, 33)-1):
```

Comment:
Isn't `pow(2, 33)` a strange power of two?

Reply:
I just copied it from the code above. The integers look 64 bit wide:
```
>>> sys.maxsize
9223372036854775807
```
So this is just an arbitrary, large number I think.

Florian, can you comment on this?

Reply:
Int in Python 3 is unbounded, so I think the value above is a rather arbitrary maximum default for this... `sys.maxsize` gives you the interpreters word-size and the maximum amount of e.g. elements in a list, but that doesn't affect ints at all.

Reply:
Okay, I'll just leave it as is.